route/build/src/lake_route.f90

```diff
@@ -91,25 +92,19 @@ SUBROUTINE lake_route(iEns,       &    ! input: index of runoff ensemble to be p
 
   ierr=0; message='lake_route/'
 
+  verbose = .false.
+  if(NETOPO_in(segIndex)%REACHIX == ixDesire)then
+    verbose = .true.
+  end if
+
     ! identify number of upstream segments of the lake being processed
     nUps = size(NETOPO_in(segIndex)%UREACHI)
 
-    allocate(fluxstate(nUps), stat=ierr, errmsg=cmessage)
```

Comment:
Nice to remove an allocate statement that wasn't really necessary.

Reply:
Thanks.  yes, this can be safely removed (no answer change)